projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.html

```diff
@@ -1,6 +1,8 @@
 <aca-page-layout>
   <div class="aca-page-layout-header">
-    <h1 class="aca-page-title">{{ 'APP.BROWSE.LIBRARIES.MENU.FAVORITE_LIBRARIES.TITLE' | translate }}</h1>
+    <h1 class="aca-page-title">
+      {{ selectedRowItemsCount < 1 ? ('APP.BROWSE.LIBRARIES.MENU.FAVORITE_LIBRARIES.TITLE' | translate) : ('APP.HEADER.SELECTED' | translate: { count: selectedRowItemsCount }) }}
```

Comment:
please see my comment here, probably can be shorter: 
https://github.com/Alfresco/alfresco-ng2-components/pull/9861#discussion_r1652566738